recipe/meta.yaml

```
@@ -1,16 +1,17 @@
 {% set name = "safetensors" %}
-{% set version = "0.4.2" %}
+{% set version = "0.4.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: acc85dcb09ec5e8aa787f588d7ad4d55c103f31e4ff060e17d92cc0e8b8cac73
+  sha256: 5fe3e9b705250d0172ed4e100a811543108653fb2b66b9e702a088ad03772a07
 build:
   number: 1
-  skip: True  # [py<38 or s390x]
+  skip: True  # [py<38]
```

```suggestion
  skip: True  # [py<37]
```
https://github.com/huggingface/safetensors/blob/v0.4.4/bindings/python/pyproject.toml#L3